frontend/appflowy_flutter/lib/mobile/presentation/bottom_sheet/bottom_sheet_view_item_body.dart

```diff
@@ -78,13 +81,14 @@ class MobileViewItemBottomSheetBody extends StatelessWidget {
             ),
           ],
         ),
+        const VSpace(8),
 
-        // remove from favorites
-
+        // remove from favorites/add to favorites
         BottomSheetActionWidget(
           svg: isFavorite
               ? FlowySvgs.m_favorite_selected_lg
               : FlowySvgs.m_favorite_unselected_lg,
+          iconColor: isFavorite ? Colors.yellow : null,
```

Comment:
Is it necessary to redefine the color?

Reply:
I think we shouldn't rely on any original color from the SVG file if it serves as an icon. The solution should be setting the icon color based on the current theme. I put `Colors.yellow` for now since we haven't formed a holistic theme file. 
Before:
<img width="494" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/14248245/2574ba6d-db72-42df-adfa-a2d7daa6bf01">
After: 
<img width="374" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/14248245/a9f7da8f-9165-45a4-a5d0-beffbd169e49">